src/content/fr/2019/css.md

```diff
@@ -0,0 +1,735 @@
+
+{{ figure_markup(
+  caption="Pourcentage de pages utilisant des propriétés personnalisées.",
+  content="5&nbsp;%",
+  classes="big-number"
+)
+}}
+
+Nous avons eu l’impression que c’était une information passionnante, car elle montre une croissance saine de l’un de nos ajouts CSS préférés. Elles étaient disponibles dans tous les principaux navigateurs depuis 2016 ou 2017, il est donc juste de dire qu’elles sont assez récentes. De nombreuses personnes sont encore en train de passer de leurs variables de préprocesseur CSS aux propriétés personnalisées CSS. Nous estimons qu’il faudra encore quelques années avant qu’elles ne deviennent la norme.
+
+## Sélecteurs
+
+### Sélecteurs via ID vs class 
+
+CSS a plusieurs façons pour cibler des éléments dans la page, alors opposons donc les ID et les classes pour voir ce qui est le plus répandu&nbsp;! Les résultats ne devraient pas être trop surprenants&nbsp;: les classes sont plus populaires&nbsp;!
+
+{{ figure_markup(
+  image="fig7.png",
+  caption="Popularité des types de sélecteurs par page.",
+  description="Diagramme à barres montrant l’adoption des sélecteur par ID et par classe. Les classes sont utilisées sur 95&nbsp;% des pages de bureau et mobiles. Les identifiants sont utilisés sur 89&nbsp;% des pages de bureau et 87&nbsp;% des pages mobiles.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQO5CabwLwQ5Lj1_9bbEFnFM1qEqCorymaBHrcaNiMSJ7sYDKHUI5iish5VAS-SxN447UTW-1-5-OjE/pubchart?oid=1216272563&format=interactive"
+  )
+}}
+
+Un bon tableau de suivi tel que celui qui suit, montrant que les classes occupent 93&nbsp;% des sélecteurs trouvés dans une feuille de style.
+
+{{ figure_markup(
+  image="fig8.png",
+  caption="Popularité des types de sélecteurs, par sélecteur.",
+  description="Diagramme à barres montrant que 94&nbsp;% des sélecteurs incluent le sélecteur de classe pour ordinateur de bureau et mobile, tandis que 7&nbsp;% des sélecteurs de bureau incluent le sélecteur par ID (8&nbsp;% pour mobile).",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQO5CabwLwQ5Lj1_9bbEFnFM1qEqCorymaBHrcaNiMSJ7sYDKHUI5iish5VAS-SxN447UTW-1-5-OjE/pubchart?oid=351006989&format=interactive"
+  )
+}}
+
+### Sélecteurs par attributs
+
+CSS a des sélecteurs de comparaison très puissants. Ce sont des sélecteurs comme `[target="_blank"]`, `[attribute^="value"]`, `[title~="rad"]`, `[attribute$="-rad"]` ou `[attribut*="valeur"]`. Les utilisez-vous&nbsp;? Vous pensez qu’ils sont beaucoup utilisés&nbsp;? Comparons comment ceux-ci sont utilisés avec les ID et les classes sur le Web.
+
+{{ figure_markup(
+  image="fig9.png",
+  caption="Popularité des opérateurs par sélecteur d’attribut ID.",
+  description="Diagramme à barres montrant la popularité des opérateurs utilisés par les sélecteurs d’attributs d’ID. Environ 4&nbsp;% des pages de bureau et mobiles utilisent «`*=`» et «`^=`». 1&nbsp;% des pages utilisent «`=`» et «`$=`». 0&nbsp;% utilise «`~=`».", 
```

Comment:
For some reason Showdown does NOT like the guillemets here and so won't generate the chapter. I can't figure out why. Do we really need them given they are in `code` styling?

```suggestion
  description="Diagramme à barres montrant la popularité des opérateurs utilisés par les sélecteurs d’attributs d’ID. Environ 4&nbsp;% des pages de bureau et mobiles utilisent `*=` et `^=`. 1&nbsp;% des pages utilisent `=` et `$=`. 0&nbsp;% utilise `~=`.", 
```

Reply:
I agree, I'm not sure that the chevrons+code pattern brings any benefit. It's also used before in the chapter.